x-pack/platform/packages/shared/file-upload-common/src/constants.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const OPEN_FILE_UPLOAD_LITE_ACTION = 'openFileUploadLiteTrigger';
```

Comment:
I have a general question about implementation. Why is the implementation using uiActions?

uiActions are designed to allow plugins to extend functionality. For example, the panel context menu for dashboard panels.

Why is uiAction action needed? What problem is being solved by using uiActions?

Reply:
>What problem is being solved by using uiActions?

Potentially dependency problems. uiActions allow any plugin to trigger this flyout easily without needing a dependency on the data_visualizer plugin. 
If there is a more correct way to do this, I can look to change it.

Reply:
When I added the a flyout to for pattern analysis to Discover, I was advised to use uiActions to trigger it. I followed the same idea for this flyout.

Reply:
Thanks for clarifying. That makes sense